From 0fcff56973ba4d88520444133d5117b2cf8bbea4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 7 Oct 2016 10:04:44 -0700
Subject: [PATCH] PUBAPI-1331 cloudapi dep on keyapi uses
 node-sdc-clients.git#master which has a major ver bump that breaks it

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index fb2eee9..bf0a4ee 100644
--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
         "assert-plus": "1.0.0",
         "asn1": "0.1.11",
         "ctype": "0.5.2",
-        "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#f382a4",
+        "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#c30dd2710ad2175095dc0e96479686fa774b8063",
         "aperture": "git+ssh://git@github.com:joyent/node-aperture.git#016977",
         "mahi": "git+ssh://git@github.com:joyent/node-mahi.git#7c0499",
         "aperture-config": "git+ssh://git@github.com:joyent/aperture-config.git#master",
-- 
2.21.0

